### PR TITLE
made index variables in the c layer for better readability, code re…

### PR DESCRIPTION
**Pull request by christopherjwang**
…formatted in c layer as well, throw_error code style consistency

### CODE REVIEW

**Comment by christopherjwang** [comment]
primarily a readability commit, and preparation for moving around index on stack for the next PR

File: c_wrapper/lua-mongoc-client.c [hunk 1]
```diff
@@ -60,7 +60,7 @@ lua_mongo_client_get_database_names(lua_State *L)
             lua_pushstring(L, database_names[i]);
             lua_settable(L, -3);
         }
-        bson_strfreev (database_names);
+        bson_strfreev(database_names);
```
Comment (ajdavis) [inline c_wrapper/lua-mongoc-client.c]: This is enforcing the opposite style =)

File: c_wrapper/lua-mongoc-collection.c [hunk 2]
```diff
@@ -72,7 +74,7 @@ lua_mongo_collection_drop (lua_State *L)
  */
 
 int
-lua_mongo_collection_count (lua_State *L)
+lua_mongo_collection_count(lua_State *L)
```
Comment (ajdavis) [inline c_wrapper/lua-mongoc-collection.c]: Same, removes a space where you intend to add one.

File: c_wrapper/lua-mongoc-collection.c [hunk 3]
```diff
@@ -82,38 +84,43 @@ lua_mongo_collection_count (lua_State *L)
...
+                                    MONGOC_QUERY_NONE, &query,
+                                    skip, limit, NULL, &error);
 
     if (count < 0) {
         throw_error = true;
         goto DONE;
     }
 
-DONE:
+    DONE:
```
Comment (ajdavis) [inline c_wrapper/lua-mongoc-collection.c]: Sigh: https://youtrack.jetbrains.com/issue/CPP-204

**Comment by ajdavis** [comment]
Couple of places where this *decreases* the style consistency with the C Driver's style, but that's super-trivial. If you feel like continuing to fix whitespace issues, go ahead, but it's ok if you don't want to spend time on that, too. I'll merge this to unblock your substantial work.

**Comment by christopherjwang** [comment]
I'd be happy to do a redo. Do you have a CLion project settings thing I could steal, and are the comments mostly inclusive on decreasing style trends or are there more?

**Comment by ajdavis** [comment]
I wouldn't worry about the tiny whitespace inconsistencies, forge ahead on 
substantial work!

On Tue, Oct 13, 2015 at 10:48 AM, christopherjwang <notifications@github.com

**Comment by christopherjwang** [comment]
Alright. Don't baby me too much. I'd like to keep this project clean if possible.

**Pull request merged**